### DISCUSSION TITLE
bringing more than 1 template into an app

### DISCUSSION THREAD

**Discussion by jastorjastor** [Budibase/budibase#10187] [Help] [answered]
Hello. 
I'd like to create an app that leverages more than 1 of the existing templates. 
Is there any documentation on this? 
Thank you.

**Comment by dewdeded** [comment, marked as answer]
I don't think that is currently possible. You have to recreate the stuff you like from other app templates in the app you develope. You can't copy pages/screen between apps.

**Reply by jastorjastor** [reply]
Thank you. I was hoping that if each database shared values then the possiblity of joining would exist. I'll be poking at this and will make updates here.
